src/ExploreQueryEditor.tsx

```diff
@@ -373,6 +406,15 @@ export default function ExploreQueryEditor(props: Props) {
           />
         </InlineField>
       </div>
+      <div className="query-field">
+        <InlineField label="logcli" tooltip="logcli command, copy and run it in the command line">
```

Comment:
I suspect people will be more confused if you display this textbox LOL

Reply:
I think so as well, will refine the wording soon.

Reply:
Is it possible to hide this textbox? We can just provide a button named "Copy log export command".

Reply:
Sure, but maybe the user wants to modify the command before running it, for example, enlarge the limit values (default is 100000)?

Reply:
After discussion, the text box will be removed.